downloads/downloads-impl/src/test/java/com/duckduckgo/downloads/impl/DownloaderUtilTest.kt

```diff
@@ -191,26 +191,102 @@ class DownloaderUtilTest {
 
     @Test
     fun whenContentDispositionWithAttachmentAndFilenameBetweenQuotesAndSpaceAfterSemicolonThenCorrectFilenameExtracted() {
-        assertEquals("filename.jpg", DownloaderUtil.parseContentDisposition("""attachment; filename="filename.jpg""""))
+        assertEquals("filename.jpg", DownloaderUtil.fileNameFromContentDisposition("""attachment; filename="filename.jpg""""))
     }
 
     @Test
     fun whenContentDispositionWithAttachmentAndFilenameBetweenQuotesAndSpacesBeforeAndAfterSemicolonThenCorrectFilenameExtracted() {
-        assertEquals("filename.jpg", DownloaderUtil.parseContentDisposition("""attachment ; filename="filename.jpg""""))
+        assertEquals("filename.jpg", DownloaderUtil.fileNameFromContentDisposition("""attachment ; filename="filename.jpg""""))
     }
 
     @Test
     fun whenContentDispositionWithAttachmentAndFilenameBetweenQuotesAndNoSpacesThenCorrectFilenameExtracted() {
-        assertEquals("filename.jpg", DownloaderUtil.parseContentDisposition("""attachment;filename="filename.jpg""""))
+        assertEquals("filename.jpg", DownloaderUtil.fileNameFromContentDisposition("""attachment;filename="filename.jpg""""))
     }
 
     @Test
     fun whenContentDispositionWithAttachmentAndFilenameWithoutQuotesAndWithSpaceAfterSemicolonThenCorrectFilenameExtracted() {
-        assertEquals("filename.jpg", DownloaderUtil.parseContentDisposition("""attachment; filename=filename.jpg"""))
+        assertEquals("filename.jpg", DownloaderUtil.fileNameFromContentDisposition("""attachment; filename=filename.jpg"""))
     }
 
     @Test
     fun whenContentDispositionWithInlineAndFilenameBetweenQuotesAndSpaceAfterSemicolonThenCorrectFilenameExtracted() {
-        assertEquals("filename.jpg", DownloaderUtil.parseContentDisposition("""inline; filename="filename.jpg""""))
+        assertEquals("filename.jpg", DownloaderUtil.fileNameFromContentDisposition("""inline; filename="filename.jpg""""))
+    }
+
+    @Test
+    fun whenContentDispositionWithInlineAndFilenameEncodedThenEncodedFilenameExtracted() {
```

Comment:
All tests have "ContentDispositionWithInline" in their names, however the actual tests use "attachment" in the content-disposition.
Either update the names or replace the "attachment" with "inline" in the content-disposition.

Reply:
Shall I duplicate all the tests that I created to cover both attachment and inline?

Reply:
No all, but have one with inline.